client/support/support-user/index.jsx

```diff
@@ -26,6 +27,10 @@ const SupportUser = React.createClass( {
 	},
 
 	onKeyboardShortcut: function( e ) {
+		if ( ! this.props.isEnabledForUser ) {
+			return false;
```

Comment:
I was thinking it would cancel event propagation, but actually we don't want to cancel it. Removed in e401b49